webpack/assets/javascripts/react_app/components/DeleteMessageDialog/DeleteConfirmationDialog.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MessageDialog, Icon, Button } from 'patternfly-react';
+import { translate as __ } from '../../common/I18n';
+import { noop } from '../../common/helpers';
+import './deleteConfirmationDialog.scss';
+
+const DeleteConfirmationDialog = ({
+  id,
+  name,
+  controller,
+  show,
+  processing,
+  onDelete,
+  onHide,
+}) => (
+  <MessageDialog
+    show={show}
+    onHide={onHide}
+    footer={
+      <React.Fragment>
+        <Button bsStyle="default" onClick={onHide} disabled={processing}>
+          {__('Cancel')}
+        </Button>
+        <Button
+          bsStyle="danger"
+          onClick={() => onDelete(controller, id)}
+          disabled={processing}
```

Comment:
would be nice to show a small spinner when `processing` is true

Reply:
Done!